Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM_test.py

```diff
@@ -0,0 +1,873 @@
+from http import HTTPStatus
+from urllib.parse import urljoin
+from FortinetFortiwebVM import ClientV1, ClientV2, Client
+from CommonServerPython import *
+import json
+import pytest
+import os
+import demistomock as demisto
+
+
+def load_mock_response(file_name: str) -> str:
+    """
+    Load mock file that simulates an API response.
+    Args:
+        file_name (str): Name of the mock response JSON file to return.
+    Returns:
+        str: Mock file content.
+    """
+    file_path = os.path.join('test_data', file_name)
+    with open(file_path, mode='r', encoding='utf-8') as mock_file:
+        return json.loads(mock_file.read())
+
+
+@pytest.fixture(autouse=True)
+def mock_client(version: str) -> Client:
+    """Create a test client for v1/v2
+
+    Args:
+        version (str): Version (V1/V2)
+
+    Returns:
+        Client: Fortieweb VM Client.
+    """
+    client_class = ClientV1 if version == ClientV1.API_VER else ClientV2
+    client: Client = client_class('http://1.1.1.1/', 'api_key', version, True, False)  # type: ignore
+    return client
+
+
+@pytest.mark.parametrize(
+    ('version', 'endpoint', 'args', 'jsonpath', 'expected_key', 'expected_value', 'status_code', 'assert_flag'), (
+        (ClientV1.API_VER, 'ServerObjects/ProtectedHostnames/ProtectedHostnames', {
+            'name': 'check'
+        }, 'protected_hostname/v1_success.json', 'name', 'check', HTTPStatus.OK, False),
+        (ClientV1.API_VER, 'ServerObjects/ProtectedHostnames/ProtectedHostnames', {
+            'name': 'check'
+        }, 'protected_hostname/v1_failed_exist.json', 'name', 'check', HTTPStatus.INTERNAL_SERVER_ERROR, True),
+        (ClientV2.API_VER, 'server-policy/allow-hosts', {
+            'name': 'check'
+        }, 'protected_hostname/v2_success.json', 'name', 'check', HTTPStatus.OK, False),
+        (ClientV2.API_VER, 'server-policy/allow-hosts', {
+            'name': 'check'
+        }, 'protected_hostname/v2_failed_exist.json', 'name', 'check', HTTPStatus.INTERNAL_SERVER_ERROR, True),
+    ))
+def test_protected_hostname_create_command(requests_mock, mock_client, version, endpoint, args, jsonpath, expected_key,
+                                           expected_value, status_code, assert_flag):
+    """
+    Scenario: Create a protected hostname group.
+    Given:
+     - User has provided correct parameters.
+     - User has provided exist name.
+    When:
+     - fortiwebvm-protected-hostname-group-create called.
+    Then:
+     - Ensure that protected hostname created.
+     - Ensure relevant error raised.
+    """
+    from FortinetFortiwebVM import protected_hostname_create_command
+    json_response = load_mock_response(jsonpath)
+    url = urljoin(mock_client.base_url, endpoint)
+    requests_mock.post(url=url, json=json_response, status_code=status_code)
+    try:
+        result = protected_hostname_create_command(mock_client, args)
+        assert expected_value in result.readable_output
```

Comment:
please check for the exact value, rather than `in`, here and in all other tests.